cogs/utility.py

```diff
@@ -674,20 +674,40 @@ async def ping(self, ctx):
 
     @commands.command()
     @checks.has_permissions(PermissionLevel.ADMINISTRATOR)
-    async def mention(self, ctx, *mention: Union[discord.Role, discord.Member]):
+    async def mention(self, ctx, *mention: Union[discord.Role, discord.Member, str]):
         """
         Change what the bot mentions at the start of each thread.
 
         Type only `{prefix}mention` to retrieve your current "mention" message.
+        `{prefix}mention disable` to disable mention.
+        `{prefix}mention reset` to reset it to default value.
         """
-        # TODO: ability to disable mention.
         current = self.bot.config["mention"]
-
         if not mention:
             embed = discord.Embed(
                 title="Current mention:", color=self.bot.main_color, description=str(current)
             )
+        elif (
+            len(mention) == 1
+            and isinstance(mention[0], str)
+            and mention[0] in ["disable", "reset"]
```

Comment:
```suggestion
            and mention[0].lower() in ["disable", "reset"]
```

Reply:
Thanks for the review. With that, I may need to change var `option = ...` to `option = mention[0].lower()` as well.